Northstar.Custom/keyvalues/scripts/weapons/mp_weapon_doubletake.txt

```diff
@@ -0,0 +1,751 @@
+WeaponData
```

Comment:
Adds a new vanilla file. @EnderBoy9217 from which VPK is that from?

Reply:
This should not be the vanilla file copied in (but it is)
@EnderBoy9217 please use keyvalues properly, they are documented

Reply:
> Adds a new vanilla file. @EnderBoy9217 from which VPK is that from?

englishclient_mp_common.bsp.pak000_dir.vpk\scripts\weapons

Reply:
@ASpoonPlaysGames Removed the vanilla file data, tested and still working